### ISSUE TITLE
@Mention Suggestion Box Behavior

### ISSUE THREAD

**Issue by jterps08** [PowerlineApp/powerline-mobile#433]
When a user starts to type an @mention, the suggestion box correctly appears and lists suggestions.

However, when user selects an item (e.g. @JTest1), the text only stays in the post/petition/comment IF the user provides a space or another character in the text field after selecting the item from the drop down. If no additional character is provided, the suggested item is not committed to the text field. 

For example, user types @JT...
System suggestions "@JTest1" etc.
User selects @JTest1 and hits send - the post/petition/comment will only show @JT (incorrect)
If user selects @Jtest1 from suggestion box and enters a space afterward and hits send, the post/petition/comment will show the full @JTest1 and the @mention works correctly. 

Screenshot TBD

**Comment by jterps08** [comment]
First I start typing the name
![screenshot_20161130-093325](https://cloud.githubusercontent.com/assets/7582457/20756623/58d65b0e-b6e1-11e6-85a9-cdb0891012d6.png)

Then I select an option from the suggestion pop-up...
![screenshot_20161130-093332](https://cloud.githubusercontent.com/assets/7582457/20756646/6cf6af80-b6e1-11e6-8e68-4c430f44ce89.png)

There is NO SPACE or any other character entered after I have selected from the drop-down and I tap the Send button... 
![screenshot_20161130-093332](https://cloud.githubusercontent.com/assets/7582457/20756840/3e55cb74-b6e2-11e6-8a8c-7957570ec8da.png)

The app seems to only have picked up the original characters I typed (before I selected from the drop down), instead of the full user ID from the drop down. As such, the @mention does not work. 
![screenshot_20161130-093342](https://cloud.githubusercontent.com/assets/7582457/20756898/6a6b7010-b6e2-11e6-97eb-7b6cd01974b3.png)

If I add a space after I've selected from the drop down, no problem.
